### PR TITLE
Fix app name casing in UI

### PR DESCRIPTION
**Pull request by sidevesh**
Added DISPLAY_NAME which is just `solaar` with first letter capitalized, as should app names be.
Didn't add this to translation stuff since I see that `Solaar` is kept as is in all translations, so won't make a difference.

### CODE REVIEW

File: lib/solaar/__init__.py [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import sys as _sys
 
 NAME = "solaar"
+DISPLAY_NAME = "Solaar"
```
Comment (pfps) [inline lib/solaar/__init__.py:22]: The display name should be derived from the name, not a separate constant.  Even better, use only Solaar and then .lower() where a lower-case name is needed.   There are already several places where .lower() is used.

**Comment by sidevesh** [comment]
@pfps made the changes, NAME is now `Solaar` and I am using `.lower()` wherever `solaar` is needed.

**Comment by pfps** [comment]
Looks good.

**Review by pfps — APPROVED** [review]
Looks good

**Pull request merged**